### PR TITLE
add sql validation for table name

### LINKED ISSUE #87: Possible sql injection
https://github.com/ongr-io/ConnectionsBundle/blob/65275acb568c16385e0f8d6ed263ef5a66eeb50f/Sync/DiffProvider/SyncJobs/JobsCleanupService.php#L75

### PR DESCRIPTION
**Pull request by tfe2012**
Closes #87

### CODE REVIEW

File: Sync/DiffProvider/SyncJobs/JobsCleanupService.php [hunk 1]
```diff
@@ -48,7 +49,7 @@ class JobsCleanupService
     public function __construct($connection, $tableName = 'ongr_sync_jobs', array $shops = [])
     {
         $this->connection = $connection;
-        $this->tableName = $tableName;
+        $this->tableName = SqlValidator::validateTableName($tableName);
```
Comment (MindaugasR) [inline Sync/DiffProvider/SyncJobs/JobsCleanupService.php]: for that it is best practice to use setter and getter

File: Sync/DiffProvider/SyncJobs/TableManager.php [hunk 2]
```diff
@@ -51,7 +53,7 @@ public function __construct($connection, $tableName = 'ongr_sync_jobs', array $s
         }
 
         $this->connection = $connection;
-        $this->tableName = $tableName;
+        $this->tableName = SqlValidator::validateTableName($tableName);
```
Comment (MindaugasR) [inline Sync/DiffProvider/SyncJobs/TableManager.php]: same here

File: Tests/Unit/Sync/JobsCleanupServiceTest.php [hunk 3]
```diff
@@ -116,12 +117,15 @@ public function testInvalidTableName($tableName)
      */
     public function getTableNameData()
     {
-        // Case #0 incorrect table name.
+        // Case #0 .
```
Comment (tautrimas) [inline Tests/Unit/Sync/JobsCleanupServiceTest.php]: Definition is necessary here and 2 below. It should describe initial condition and expected test outcome.

**Comment by trylika** [comment]
Closes #87

File: Sync/SqlValidator.php [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    /**
+     * Validate table name.
+     *
+     * @param string $tableName
+     *
+     * @throws InvalidArgumentException
+     *
+     * @return string
+     */
+    public static function validateTableName($tableName)
```
Comment (MindaugasR) [inline Sync/SqlValidator.php]: is there an unit tests for this method?

Comment (tfe2012) [inline Sync/SqlValidator.php]: Tests/Unit/Sync/JobsCleanupServiceTest.php

**Pull request merged**